docs/introduction/Examples.md

```diff
@@ -1 +1,137 @@
-## Examples
\ No newline at end of file
+## Examples
+### Simple - reading from an existing xlsx spread sheet.
+The following C plus plus code will read the values from an xlsx file and print the string values to the screen. This is a very simple example to get you started.
+
+```
+#include <iostream>
+#include <xlnt/xlnt.hpp>
+
+using namespace std;
+using namespace xlnt;
+int main()
+{
+    workbook wb;
+    wb.load("/home/timothymccallum/test.xlsx");
+    auto ws = wb.active_sheet();
+    clog << "Processing spread sheet" << endl;
+    for (auto row : ws.rows(false)) 
+	{ 
+	    for (auto cell : row) 
+	        { 
+		    clog << cell.to_string() << endl;
+		}
+	}
+    clog << "Processing complete" << endl;
+    return 0;
+}
+```
+Save the contents of the above file 
+```
+/home/timothymccallum/process.cpp
+```
+Compile by typing the following command
+```
+g++ -std=c++14 -Ixlnt/include -Lxlnt/lib -lxlnt process.cpp -o process
```

Comment:
No extra directories were created during the installation of xlnt. I just successfully ran the following command.
```
g++ -std=c++14 -lxlnt process.cpp -o process
```
 I just copied that command style from the xlnt GitHub home page < https://github.com/tfussell/xlnt/ > (the commented out code "// compile with -std=c++14 -Ixlnt/include -Lxlnt/lib -lxlnt" below the heading "Example").